src/backend/base/langflow/services/telemetry/service.py

```diff
@@ -82,7 +82,7 @@ async def log_package_run(self, payload: RunPayload):
         await self._queue_event((self.send_telemetry_data, payload, "run"))
 
     async def log_package_shutdown(self):
-        payload = ShutdownPayload(timeRunning=(datetime.now(timezone.utc) - self._start_time).seconds)
+        payload = ShutdownPayload(time_running=(datetime.now(timezone.utc) - self._start_time).seconds)
         await self._queue_event(payload)
```

Comment:
Shouldn't it go through `self.send_telemetry_data` ?

Reply:
Wow.. I think it should. Good catch.